wolfssl/internal.h

```diff
@@ -1176,6 +1176,14 @@ enum {
     #define MAX_EARLY_DATA_SZ  4096
 #endif
 
+#ifndef WOLFSSL_MAX_RSA_BITS
+    #if (defined(USE_FAST_MATH) && defined(FP_MAX_BITS) && FP_MAX_BITS >= 16384)
+        #define WOLFSSL_MAX_RSA_BITS 8192
```

Comment:
Although 8k is a bit crazy already - shouldn't this be:
<pre>
#define WOLFSSL_MAX_RSA_BITS  (FP_MAX_BITS / 2 )
</pre>
For the even crazier people that want even bigger.

Reply:
Thanks @grooverdan!
Fixed.